### ISSUE TITLE
Dockerfile: Build fails due to http redirect from blender.org

### ISSUE THREAD

**Issue by zeta0134** [zeta0134/pikmin-nds#33]
When installing from the current dockerfile, the build fails with the following output during the blender install step:

```
Step 6/17 : RUN curl -O http://download.blender.org/release/Blender2.74/blender-2.74-linux-glibc211-x86_64.tar.bz2 &&   tar xf blender-2.74-linux-glibc211-x86_64.tar.bz2 &&   mv blender-2.74-linux-glibc211-x86_64 /opt/ &&   rm blender-2.74-linux-glibc211-x86_64.tar.bz2 &&   echo '/opt/blender-2.74-linux-glibc211-x86_64/lib' > /etc/ld.so.conf.d/blender.conf &&   ldconfig
 ---> Running in 7d493ab53164
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   162  100   162    0     0    273      0 --:--:-- --:--:-- --:--:--   273
tar: This does not look like a tar archive
bzip2: (stdin) is not a bzip2 file.
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

**Comment by zeta0134** [comment]
This occurs because blender is now issuing an http -> https redirect:
```
[zeta0134@msi-antergos pikmin-nds]$ curl -I http://download.blender.org/release/Blender2.74/blender-2.74-linux-glibc211-x86_64.tar.bz2
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Sat, 08 Feb 2020 05:54:27 GMT
Content-Type: text/html
Content-Length: 162
Connection: keep-alive
Location: https://download.blender.org/release/Blender2.74/blender-2.74-linux-glibc211-x86_64.tar.bz2
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
```

curl could be configured to follow the redirect with -L, but we should probably prefer the https link instead.

**Comment by zeta0134** [comment]
Closed by #34
